anti_patterns/deny-warnings.md

```diff
@@ -0,0 +1,101 @@
+# `#![deny(warnings)]`
+
+## Description
+
+A well-intentioned crate author wants to ensure their code builds without
+warnings. So they annotate their crate root with
+
+## Example
+
+```rust
+#![deny(warnings)]
+
+// All is well.
+```
+
+## Advantages
+
+It is short and will stop the build if anything is amiss.
+
+## Drawbacks
+
+By disallowing the compiler to build with warnings, a crate author opts out of
+Rust's famed stability. Sometimes new features or old misfeatures need a change
+in how things are done, thus lints are written that `warn` for a certain grace
+period before being turned to `deny`. Sometimes APIs get deprecated, so their
+use will emit a warning where before there was none.
+
+ALl this conspires to potentially break the build whenever something changes –
+not only for the original developer, but also for those who wish to use their
+code.
+
+Furthermore, crates that supply additional lints (e.g. [rust-clippy]) can no
+longer be used unless the annotation is removed.
+
+## Alternatives
+
+luckily, there are two ways of tackling this problem: First, we can decouple
```

Comment:
I think the word "luckily" should be removed. These articles tend to have a sparse writing style, which discourages extra words like this.